admin-manual/security/security.rst

```diff
@@ -207,6 +207,39 @@ Shibboleth backend configuration in Storage Service
 
 #. Restart the Storage Service.
 
+.. _oidc-setup:
+
+OIDC setup
+==========
+
+`OIDC <oidc_>`_ (OpenID Connect) is an identity layer on top of OAuth 2.0, providing
+identity verification, authentication and basic information about the end user. This
+feature allows administrators to deploy both the Archivematica Dashboard and Storage
+Service Django applications with Shibboleth authentication.
```

Comment:
```suggestion
Service Django applications with OIDC authentication.
```

Reply:
@alexwlchan Is this the only change you see as needed? I can get one of our devs to review and then we can rebase and merge 🎉